main.js

```diff
@@ -106,56 +113,62 @@ function toggleRulerCommand() {
 }
 
 app.dock.hide();
-app.on('ready', function() {
+app.on('ready', function () {
 	let trayIcon = new Tray(path.join(__dirname, 'src/assets/images/lrTemplate.png'));
 
 	var trayMenuTemplate = [
-			{
-					label: 'New ruler',
-					accelerator: 'Command+Ctrl+R',
-					click: createNewRuler
-			},
-			{
-					label: 'Toggle rulers',
-					accelerator: 'Command+Ctrl+T',
-					click: toggleRulerCommand,
-					enabled: false
-			},
-			{
-				type: 'separator'
-			},
-			{
-				label: 'Settings',
-				click: () => {
-					if (settingsWindow) {
-						return;
-					}
-
-					settingsWindow = new BrowserWindow({
-						width: 200,
-						height: 170,
-						alwaysOnTop: true
-					});
-
-					settingsWindow.loadURL(`file://${__dirname}/src/app/settings/settings.html`);
-
-					settingsWindow.on('closed', () => {
-						settingsWindow = null;
-					});
+		{
+			label: 'New Ruler',
+			accelerator: 'Command+Ctrl+R',
+			click: createNewRuler
+		},
+		{
+			label: 'Toggle Rulers',
+			accelerator: 'Command+Ctrl+T',
+			click: toggleRulerCommand,
+			enabled: false
+		},
+		{
+			label: 'Toggle Center Guides',
+			accelerator: 'Command+;',
+			click: toggleCenterGuidesCommand,
+			enabled: true
```

Comment:
It would be possible, if we scaffolded out a state-machine to pub-sub to specific app-level events and then redraw the menu accordingly. IMO, it a state-machine would only be justified if we were to build out more features that could benefit from subscribing to state changes (like various react components, or more interactive features embedded into the ruler), so we could put a feature like that on the V2 roadmap if a state machine makes sense.